locale/pt-br/texts.po

```diff
@@ -21,25 +30,337 @@ msgstr ""
+
+#: src/screens/CreateToken.js:146 src/screens/CreateTokenName.js:51
+msgid "Token Name"
+msgstr "Nome do Token"
+
+#: src/screens/CreateToken.js:153
+msgid "Token Symbol"
+msgstr ""
+
+#: src/screens/CreateToken.js:161 src/screens/PaymentRequestDetail.js:108
+msgid "Amount"
+msgstr "Quantidade"
+
+#: src/screens/CreateToken.js:172
+msgid "Create New Token"
+msgstr "Criar Novo Token"
+
+#: src/screens/CreateTokenAmount.js:109 src/screens/CreateTokenConfirm.js:126
+#: src/screens/CreateTokenDepositNotice.js:39 src/screens/CreateTokenName.js:44
+#: src/screens/CreateTokenSymbol.js:63
+msgid "CREATE TOKEN"
+msgstr "CRIAR TOKEN"
+
+#: src/screens/CreateTokenAmount.js:117
+#, javascript-format
+msgid "Amount of ${ this.name } (${ this.symbol })"
+msgstr "Quantidade de ${ this.name } (${ this.symbol })"
+
+#: src/screens/CreateTokenAmount.js:129
+msgid "Deposit:"
+msgstr "Depósito:"
+
+#: src/screens/CreateTokenAmount.js:133
+#, javascript-format
+msgid "You have ${ amountAvailableText } HTR available"
+msgstr "Você tem ${ amountAvailableText } HTR disponíveis"
+
+#: src/screens/CreateTokenAmount.js:138 src/screens/CreateTokenName.js:65
+#: src/screens/CreateTokenSymbol.js:85 src/screens/InitWallet.js:208
+#: src/screens/InitWallet.js:328 src/screens/SendAddressInput.js:66
+#: src/screens/SendAmountInput.js:159
+msgid "Next"
+msgstr "Próximo"
+
+#.  show loading modal
+#.  eslint-disable-next-line react/jsx-indent
+#: src/screens/CreateTokenConfirm.js:66
+msgid "Creating token"
+msgstr "Criando um token"
+
+#: src/screens/CreateTokenConfirm.js:135
+#, javascript-format
+msgid "Amount of ${ this.name }"
+msgstr "Quantidade de ${ this.name }"
+
+#: src/screens/CreateTokenConfirm.js:143
+msgid "Token name"
+msgstr "Nome do token"
+
+#: src/screens/CreateTokenConfirm.js:149 src/screens/CreateTokenSymbol.js:70
+msgid "Token symbol"
+msgstr "Símbolo do token"
+
+#: src/screens/CreateTokenConfirm.js:155
+msgid "Deposit"
+msgstr "Depósito"
+
+#: src/screens/CreateTokenConfirm.js:162
+msgid "Create token"
+msgstr "Criar um token"
+
+#: src/screens/CreateTokenDepositNotice.js:45
+msgid "When creating new tokens, a "
+msgstr "Quando um novo token é criado, um "
+
+#: src/screens/CreateTokenDepositNotice.js:47
+#, javascript-format
+msgid "deposit of ${ deposit_percentage }% "
+msgstr "depósito de ${ deposit_percentage }% "
+
+#: src/screens/CreateTokenDepositNotice.js:49
+msgid ""
+"in HTR is required - e.g. if you create 1000 NewCoins, 10 HTR are needed as "
+"deposit."
+msgstr ""
+"é exigido em HTR - por exemplo, se você criar 1.000 MinhaMoeda, 10 HTR serão "
+"exigidos como depósito."
+
+#: src/screens/CreateTokenDepositNotice.js:52
+msgid ""
+"If these tokens are later melted, the HTR deposit will be returned. Read "
+"more about it "
+msgstr ""
+"Se esses tokens forem destruídos futuramente, o depósito em HTR será "
+"devoldido. Leia mais sobre esse depósito "
+
+#: src/screens/CreateTokenDepositNotice.js:53
+msgid "here"
+msgstr "aqui"
```

Comment:
This is a good place to put a translation comment explaining _what_ here refers about. I agree that we could wait to do this when we need to. But it is good to test if the tool supports comments and we how to use them.

Reply:
Fixed.